src/components/hmi_message_handler/test/mqueue_adapter_test.cc

```diff
@@ -30,65 +30,62 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include <gtest/gtest.h>
-#include <gmock/gmock.h>
+#include "gtest/gtest.h"
 
-#include "hmi_message_handler/hmi_message_handler.h"
 #include "hmi_message_handler/mqueue_adapter.h"
+#include "hmi_message_handler/mock_hmi_message_handler.h"
+
+namespace test {
+namespace components {
+namespace hmi_message_handler_test {
 
 using hmi_message_handler::MessageSharedPointer;
```

Comment:
@VProdanov , no, if we remove include of mock_hmi_mh , this file will be not builda-able and will have some errors.